_drafts/community.md

```diff
@@ -0,0 +1,73 @@
+---
+name: community
+layout: post_discourse
+title: Hey! You there! You are welcome here.
+authors:
+  - name: Shannon E. Ellis
+    url: https://shanellis.github.io/
+categories:
+  - blog
+topicid: ???
+tags:
+  - unconf
+  - community
+  - meetings
+---
```

Comment:
Thanks @stefaniebutland and @sckott for your help! Comments addressed -- feel free to change wording on the additions (or anything really!)